### PR TITLE
[WIP] Feature/websockets 

### PR DESCRIPTION
**Pull request by xx-shitai-xx**

### CODE REVIEW

File: server/src/TPlayer/TPlayer.cpp [hunk 1]
```diff
@@ -2258,6 +2318,7 @@ bool TPlayer::msgPLI_LOGIN(CString& pPacket)
 	//serverlog.out("[%s]    Key: %d\n", server->getName().text(), key);
 	serverlog.out("[%s]    Version:\t%s (%s)\n", server->getName().text(), version.text(), getVersionString(version, type));
 	serverlog.out("[%s]    Account:\t%s\n", server->getName().text(), accountName.text());
+	serverlog.out("[%s]    Password:\t%s\n", server->getName().text(), password.text());
```
Comment (xx-shitai-xx) [inline server/src/TPlayer/TPlayer.cpp:2321]:
```suggestion
```

File: server/src/TPlayer/TPlayer.cpp [hunk 2]
```diff
@@ -2287,7 +2339,7 @@ bool TPlayer::msgPLI_LOGIN(CString& pPacket)
 	if (versionID == CLVER_UNKNOWN) {
 		// Get Iterator-Key
 		// 2.19+ RC and any client should get the key.
-		if ( isClient() || (isRC() && in_codec.getGen() > ENCRYPT_GEN_3) || getKey == true ) {
+		if ( (isClient() && type != PLTYPE_WEB) || (isRC() && in_codec.getGen() > ENCRYPT_GEN_3) || getKey == true ) {
```
Comment (xtjoeytx) [inline server/src/TPlayer/TPlayer.cpp:2342]: is it right to say this shouldn't happen for anything < encrypt_gen_2 rather than use the type != web

Comment (xtjoeytx) [reply server/src/TPlayer/TPlayer.cpp:2342]: actually looking at the code, scratch that - as it is, is fine

**Review by xtjoeytx — APPROVED** [review]

**Comment by LoneBoco** [comment]
It looks fine.  There are things I would like to change but we those have to wait until Xcode improves.

**Comment by xtjoeytx** [comment]
> It looks fine. There are things I would like to change but we those have to wait until Xcode improves.

ur damn right

**Pull request merged**